wayland-display-core/src/comp/rendering.rs

```diff
@@ -1,24 +1,32 @@
 use std::time::{Duration, Instant};
 
 use smithay::{
+    desktop::space::render_output,
     backend::renderer::{
-        damage::DamageTrackedRendererError as DTRError,
+        damage::Error as DTRError,
         element::{
             memory::MemoryRenderBufferRenderElement, surface::WaylandSurfaceRenderElement,
             RenderElementStates,
         },
-        gles2::Gles2Renderer,
+        gles::GlesRenderer,
         Bind, ExportMem, ImportAll, ImportMem, Renderer, Unbind,
     },
-    desktop::space::render_output,
     input::pointer::CursorImageStatus,
     render_elements,
     utils::{Physical, Rectangle},
 };
+use smithay::backend::allocator::Fourcc;
+use smithay::backend::renderer::element::Kind;
+use smithay::backend::renderer::element::memory::MemoryBuffer;
+use smithay::reexports::drm::buffer::DrmFourcc;
+use smithay::utils::Size;
 
 use super::State;
 
-pub const CURSOR_DATA_BYTES: &[u8] = include_bytes!("../../resources/cursor.rgba");
+pub const CURSOR_DATA_BYTES: MemoryBuffer = MemoryBuffer::from_slice(
+    include_bytes!("../../resources/cursor.rgba"),
+    Fourcc::Rgba4444, // TODO: Format???
```

Comment:
Which format is the cursor? Btw, is this the right way to load it up in the new `MemoryBuffer` struct?

Reply:
Either `Rgba8888` or `Abgr8888`, I don't fully remember, probably the former given the name, but the different apis use different edianess to express the order...